CHANGELOG.md

```diff
@@ -12,6 +20,9 @@
 * Crash when quering with 'Not()' followed by empty group. (Issue [Realm Core #4168]() since v1.0.0)
 * Streaming download notifiers reported incorrect values for transferrable bytes. (Issue [Realm Core #5008]() since v11.5.2)
 
+### Breaking Changes
+* [RealmApp] The default client reset strategy has been changed from `ManuallyRecoverUnsyncedChangesStrategy`, formerly named `ClientResetHandler`, to `DiscardUnsyncedChangesStrategy`.
```

Comment:
You are right, it is not a breaking change of any code base, if they are already using a client reset it would be manual. 

I am remove it and adding your suggestion.